### DISCUSSION TITLE
Fulfillment Inbound Get Labels (Async) response model

### DISCUSSION THREAD

**Discussion by EdwardCordero** [abuzuhri/Amazon-SP-API-CSharp#175] [Q&A] [answered]
Hi I was working with the package and trying to call the get labels endpoint in fulfilment inbound and saw the response model was different from what Amazon's documentation shows. In the package it is "GetPrepInstructionsResult" I was wondering if this is an error as the package does have the documented response model "GetLabelsResponse"

Here is a picture showing the package's methods where the response is "GetPrepInstructionsResult"
![Screenshot 2022-05-02 140154](https://user-images.githubusercontent.com/48169150/166327124-412444bc-5566-487b-9cf2-e38a827af9a1.png)

The package also already has a response model called "GetLabelsResponse" it's just not being used here

**Comment by abuzuhri** [comment]
Hi @EdwardCordero 

This look like bug in my library , i will fix it soon

**Comment by abuzuhri** [comment, marked as answer]
Bug fixed with v1.3.4

**Reply by EdwardCordero** [reply]
Thank you for the quick response!
